### DISCUSSION TITLE
不需要图片索引怎么办？

### DISCUSSION THREAD

**Discussion by gongwen1205** [whutug/whu-thesis#89] [Q&A]
图片索引代码定位到\begin{document}位置，该怎么修改去掉这部分索引目录呢？
![image](https://user-images.githubusercontent.com/62161547/157792401-118b7d26-ef3f-423e-9b84-64f39e09db23.png)

**Comment by xkwxdyy** [comment]
你确定不需要是吧？下面的表格索引呢

**Reply by gongwen1205** [reply]
对的，都不需要

**Reply by xkwxdyy** [reply]
<img width="871" alt="image" src="https://user-images.githubusercontent.com/87134063/157794339-ac8a6170-894e-4d99-9658-fac2fbcffbf4.png">

有键值控制的，还请仔细阅读手册

**Comment by zepinglee** [comment]
另外一种模板的设计风格是沿用 LaTeX 标准文档类的接口，让用户自己选择调用 `\maketitle`, `\tableofcontents`, `\listoffigures` 等命令。

**Reply by xkwxdyy** [reply]
好的，以后有空慢慢处理
